konrad/radiation/arts.py

```diff
@@ -13,12 +13,13 @@
 from .rrtmg import RRTMG
 from .common import fluxes2heating
 
+from scipy.constants import speed_of_light as c
 
 logger = logging.getLogger(__name__)
 
 
 class _ARTS:
-    def __init__(self, ws=None, threads=None, nstreams=4, scale_vmr=True, verbosity=0):
+    def __init__(self, ws=None, threads=None, nstreams=4, scale_vmr=True, verbosity=0, quadrature = False, quadrature_filename=join(dirname(__file__), "data", '64_quadrature.h5'), lookup_filename=join(dirname(__file__), "data", 'quadrature_lookup.xml')):
```

Comment:
When using anything but trivial constants (e.g. `int`, `float`, `str`, `bool`) as a default, it is common practice to use `None` as a default. In your code you can then check and set the value:
```py
if quadrature_file is None:
     quadrature_filename = join(dirname(__file__), "data", '64_quadrature.h5')
```
This keeps the function signature cleaner and prevents to run code when defining the function.